src/FormidableValidations.res

```diff
@@ -27,26 +27,36 @@ module Validator = {
   type t<'values, 'value, 'error> = Args.t<'values, 'value, 'error> => Value.t<'value, 'error>
 }
 
-module Pair = {
-  type t<'values, 'value, 'error> = (list<string>, Validator.t<'values, 'value, 'error>)
+module Description = {
+  type t<'values, 'value, 'error> = {
+    names: list<string>,
+    validator: Validator.t<'values, 'value, 'error>,
+  }
 }
 
-type t<'values, 'value, 'error> = (Strategy.t, Pair.t<'values, 'value, 'error>)
+type t<'values, 'value, 'error> = (Strategy.t, Description.t<'values, 'value, 'error>)
+
+module type Values = {
+  type t
+}
```

Comment:
Feel like you are not using it

Reply:
You're right, I dropped it in the other pr 👍